server/controllers/automationController.js

```diff
@@ -205,6 +205,16 @@ const paginationData = async (req, res) => {
   return paginationResponse(res, allData, page, numberOfPages, data, nextPage, prevPage);
 };
 
+/**
+   * @desc Gets automation processes to be re-run for an on-boarding placement
+   *
+   * @param {list} freckleAutomations list of freckle processes
```

Comment:
From a JavaScript background, I guess you meant to write: {Array} instead of {list}

Reply:
Would also be nice if you capitalise the first words in the parameter descriptions

Reply:
resolved